hours/importer/base.py

```diff
@@ -224,7 +224,6 @@ def _update_or_create_object(
         object_origins = {
```

Comment:
See commit a2d20eb30abcdfaac450488a026fe0a1a370ae01 and revert this to how it was before that.

Reply:
Thanks. Fixed it. 👍